cmd/handler-api.go

```diff
@@ -45,7 +46,9 @@ func (t *apiConfig) init(cfg api.Config, setDriveCount int) {
 	if cfg.RequestsMax <= 0 {
 		stats, err := sys.GetStats()
 		if err != nil {
-			return
+			logger.LogIf(GlobalContext, err)
+			// Default to 16 GiB, not critical.
+			stats.TotalRAM = 16 << 30
```

Comment:
the reason for the return is because we don't apply any request limits if we can't get any RAM size whatsoever, I thought that this makes sense we don't know what the system is capable of. 

Setting a conservative 16GiB limit is also okay.

Reply:
It appears unintentional that the last part is skipped. At least this will make it more visible if there is a problem.

Not expecting this code to ever be executed, but just in case some OS is weird.